bats/tests/helpers/utils.bash

```diff
@@ -1,11 +1,15 @@
 is_true() {
-    # case-insensitive check; false values: '', '0', 'no', and 'false'
-    if [[ "${1,,}" =~ ^(0|no|false)?$ ]]; then
-        false
+     # case-insensitive check; false values: '', '0', 'no', and 'false'
+    #if [[ "${1,,}" =~ ^(0|no|false)?$ ]]; then
+        local value="$(echo "$1" | tr '[A-Z]' '[a-z]')"
+    #fi
+    if [[ "$value" =~ ^(0|no|false)?$ ]]; then
+         false
```

Comment:
Please remove commented out code; it still exists in the git history if you ever want to refer back to it.

Also please use consistent indentation of 4 spaces by indent level.